.travis.yml

```diff
@@ -18,25 +18,31 @@
 #
 dist: xenial
 language: python
-python:
-  - "3.6"
 env:
   global:
-    - TRAVIS_CACHE=$HOME/.travis_cache/
+    - BUILD_ID=${TRAVIS_BUILD_ID}
+    - BRANCH_NAME=${TRAVIS_BRANCH}
   matrix:
-    - TOX_ENV=py27-backend_mysql-env_docker
-    - TOX_ENV=py27-backend_sqlite-env_docker
-    - TOX_ENV=py27-backend_postgres-env_docker
-    - TOX_ENV=py35-backend_mysql-env_docker PYTHON_VERSION=3
-    - TOX_ENV=py35-backend_sqlite-env_docker PYTHON_VERSION=3
-    - TOX_ENV=py35-backend_postgres-env_docker PYTHON_VERSION=3
-    - TOX_ENV=py27-backend_postgres-env_kubernetes KUBERNETES_VERSION=v1.9.0
-    - TOX_ENV=py35-backend_postgres-env_kubernetes KUBERNETES_VERSION=v1.13.0 PYTHON_VERSION=3
-
+    - BACKEND=mysql ENV=docker
+    - BACKEND=postgres ENV=docker
+    - BACKEND=sqlite ENV=docker
+    - BACKEND=postgres ENV=kubernetes KUBERNETES_VERSION=v1.9.0
+    - BACKEND=postgres ENV=kubernetes KUBERNETES_VERSION=v1.13.0
+python:
+  - '3.6'
+  - '3.5' # TODO: remove 3.5 (or 3.6?) version from the final build
```

Comment:
3.5 is still supported, so I think it is good to leave it in for now.

Reply:
More specifically 3.5.2 might be worth testing explicitly if we can - that is what ubuntu ships and I know one of our deps had an issue with `typing` as shipped in that version.